podcasts/Sharing/SharingModal.swift

```diff
@@ -60,13 +60,18 @@ enum SharingModal {
 
         let optionPicker = OptionsPicker(title: L10n.share.uppercased(), themeOverride: .dark, colors: colors)
 
-        let actions: [OptionAction] = Option.allCases(episode: episode, podcast: podcast, currentTime: PlaybackManager.shared.currentTime()).map { option in
+        let actions: [OptionAction] = Option.allCases(episode: episode, podcast: podcast, currentTime: episode?.playedUpTo ?? 0).map { option in
```

Comment:
Why this change? is the playerUpTo different from currentTime? Just wanted to understand if this is related to this PR.

Reply:
This `SharingModal` is only used with the new sharing design so nothing has changed for our existing sharing.

This just uses the passed in episode's `playedUpTo` time instead of the player's `currentTime` because you can share a link to the current timestamp of an episode without beginning playback. (Podcast > Episode > Share > Current position)

Reply:
Thanks for the clarification.